lib/secure_headers/headers/expect_ct.rb

```diff
@@ -0,0 +1,69 @@
+module SecureHeaders
+  class ExpectCtConfigError < StandardError; end
+
+  class ExpectCt
+    HEADER_NAME = 'Expect-CT'.freeze
+    CONFIG_KEY  = :expect_ct
+    INVALID_CONFIGURATION_ERROR = 'config must be a hash.'.freeze
```

Comment:
Oh my! What caveman would ever commit files with mixed quotes?!?! 😛  Fixed in 9faf299